scripts/zkai_d128_activation_swiglu_proof_input.py

```
@@ -0,0 +1,635 @@
+
+
+def _atomic_write_text(path: pathlib.Path, text: str) -> None:
+    resolved = _assert_repo_path(path)
+    resolved.parent.mkdir(parents=True, exist_ok=True)
+    fd, tmp_name = tempfile.mkstemp(prefix=f".{resolved.name}.", suffix=".tmp", dir=resolved.parent)
+    tmp_path = pathlib.Path(tmp_name)
+    try:
+        with os.fdopen(fd, "w", encoding="utf-8", newline="") as handle:
+            handle.write(text)
+        os.replace(tmp_path, resolved)
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)